svy/src/helpers/protocol.ts

```diff
@@ -0,0 +1,15 @@
+import { Protocol } from "../../generated/schema";
+
+const protocolId = "1";
```

Comment:
move protocol id to constants 

https://github.com/savvy-finance/savvy-subgraphs/blob/main/savvy-traderjoe-2/src/constants.ts#L31